jboss/container/wildfly/s2i/bash/module.yaml

```diff
@@ -0,0 +1,7 @@
+schema_version: 1
+name: jboss.container.common.s2i.bash
```

Comment:
It seems this should be jboss.container.wildly.common.s2i.bash and move it to jboss/container/wildfly/common/s2i/bash/ ?